### PR TITLE
Add more explicit docs to expression and function docs.

### PR DESCRIPTION
**Pull request by markstory**
Refs cakephp/cakephp#7157

### CODE REVIEW

**Comment by antograssiot** [comment]
looks good to me. I added the french translation.

**Comment by HavokInspiration** [comment]
Translation looks good to me

File: en/orm/query-builder.rst [hunk 1]
```diff
@@ -359,6 +362,19 @@ Would result in::
 
     SELECT YEAR(created) as yearCreated, DATE_FORMAT(created, '%H:%i') as timeCreated FROM articles;
 
+You should remember to use the function builder whenever you need to put
+untrusted data into SQL functions or stored procedures::
+
+    // Use a stored procedure
+    $query = $articles->find();
+    $lev = $query->func()->levenshtein([$search, 'LOWER(title)' => 'literal']);
+    $query->where(function ($exp) use ($query) {
```
Comment (HavokInspiration) [inline en/orm/query-builder.rst]: Should it be ``function ($exp) use ($lev) {`` ?

Comment (markstory) [inline en/orm/query-builder.rst]: Yes it should.

**Pull request merged**